ci-operator/step-registry/rosa/terraform/e2e/teardown/rosa-terraform-e2e-teardown-commands.sh

```diff
@@ -19,6 +19,9 @@ go mod vendor
 
 EXIT_STATUS=0
 
+GATEWAY_URL="$(echo "${TF_VARS}" | grep 'url' | awk -F '=' '{print $2}' | sed 's/[ |"]//g')"
```

Comment:
having all the options bundled in one ``TF_VARS`` var (instead of compiling them in the step code) seems to over-complicate stuff with ``grep`` and ``awk`` and ``sed``. why is it desired?

Reply:
This GATEWAY_URL will be removed (after this PR is merged, and after a PR in terrafom-provider is merged).